app/views/site/contribute.erb

```diff
@@ -0,0 +1,202 @@
+          </p>
+
+          <p>
+            Is an issue no longer relevant? Can it be closed? Do we need more information? If it's a bug, is it reproducible? If it's a discussion and we need to make some sort of decision, then it should be labeled as such.
+          </p>
+
+          <p>
+            Help us get our <a href="https://github.com/exercism/exercism.io/issues">issues</a> under control.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Fix Bugs</h2>
+
+          <p>
+            Exercism is currently a collection of experiments that grew into an app with the help of duct tape, chewing gum, and chicken wire. It's a mess, and it's got bugs.
+          </p>
+
+          <p>
+            Check out the <a href="https://github.com/exercism/exercism.io/issues?q=is%3Aissue+is%3Aopen+label%3Abug">list of open bugs</a> if you want to help us troubleshoot, find reproduction steps, and fix stuff.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Add Features</h2>
+
+          <p>
+            Exercism is a collective work, we are always open to suggestions and new ideas.
+          </p>
+
+          <p>
+            Have a new idea? <a href="https://github.com/exercism/discussions/issues">Open a discussion</a> so we can push things further.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Design Stuff</h2>
+
+          <p>
+            Exercism has for the most part been designed by accident by non-designers, and it shows!
+          </p>
+
+          <p>
+            We need help at every level of design. Know how to design icons? Wonderful. Want to make the pages look consistent and line things up? Fantastic. Want to redesign an entire page or feature or section? <a href="https://github.com/exercism/discussions/issues">Open an issue</a> so we can talk through some of the hard-earned learning that we've picked up along the way.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Help Us Think</h2>
+
+          <p>
+            There's a lot that we haven't quite figured out about how to make Exercism as useful as possible.
+          </p>
+
+          <p>
+            If you like to think through usability issues, communication, design, features, roadmap, collaboration, or any other exciting open source stuff, join us in the <a href="https://github.com/exercism/discussions/issues">discussions repository</a>, where we're trying to sort out Exercism's future.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>New Languages</h2>
```

Comment:
Let's see if we can find an actionable header (so far we've been really good at headers that are essentially: **Do this!**).

Add New Languages?
Start a New Track?